app/src/main/java/com/rajatgoyal/puzzle15/util/AchievementHandler.java

```diff
@@ -0,0 +1,116 @@
+package com.rajatgoyal.puzzle15.util;
+
+import android.app.Activity;
+import android.widget.Toast;
+
+import com.google.android.gms.games.AchievementsClient;
+import com.rajatgoyal.puzzle15.R;
+
+import timber.log.Timber;
+
+/**
+ * Created by rajatgoyal715 on 20/2/19.
+ */
+public class AchievementHandler {
+
+    private AchievementsClient client;
+
+    private static int[] PLAYED_IDS = {
+            R.string.achievement_newbie,
+    };
+    private static int[] COMPLETED_IDS = {
+            R.string.achievement_novice__1_game,
+            R.string.achievement_beginner__5_games,
+            R.string.achievement_skilled__10_games,
+            R.string.achievement_proficient__15_games,
+            R.string.achievement_experienced__20_games,
+            R.string.achievement_advanced__30_games,
+            R.string.achievement_expert__50_games,
+            R.string.achievement_master__80_games,
+            R.string.achievement_legend__100_games
+    };
+
+    private static int[] TIMES = {600, 300, 120, 90, 60};
+    private static int[] TIME_IDS = {
```

Comment:
Annotate these with `@StringRes` annotation so that any other kind of int value is not accidentally added.
`private static @StringRes int[] TIME_IDS`